third_party/move/move-vm/runtime/src/runtime.rs

```diff
@@ -501,4 +536,8 @@ impl VMRuntime {
     pub(crate) fn loader(&self) -> &Loader {
         &self.loader
     }
+
+    pub(crate) fn module_storage(&self) -> Arc<dyn ModuleStorage> {
```

Comment:
where is this used?

Reply:
This is used for session creation IIRC to preserve the old api.